test/unit_Polytope.jl

```diff
@@ -350,6 +350,12 @@ if test_suite_polyhedra
         @test BallInf(N[0, 0], N(1)) ⊆ P
         @test !(BallInf(N[0, 0], N(1.01)) ⊆ P)
 
+        # concrete minkowski sum
+        B = convert(VPolytope, BallInf(N[0, 0, 0], N(1)))
+        X = minkowski_sum(B, B, backend=CDDLib.Library())
+        twoB = 2.0*B
+        @test X ⊆ twoB && twoB ⊆ X
```

Comment:
What about comparing the vertices approximately instead?

Reply:
This test passes here, i think that we can keep the double inclusion. On the other hand, without converting to `VPolytope` it fails; i reported it [here](https://github.com/JuliaReach/LazySets.jl/issues/1481).

Reply:
Can you add another test where the vertices of the two operands are not identical (to catch more bugs)?
```julia
P1 = VPolytope([N[0, 0], N[0, 1]]
P2 = VPolytope([N[0, 0], N[1, 0]]
Q = minkowski_sum(P1, P2)
@test ispermutation(vertices_list(Q), [N[0, 0], N[0, 1], N[1, 0], N[1, 1]])
```